package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "rollup --config",
-    "release": "pnpm build && changeset publish --dry-run"
```

Comment:
👇 The `bump-version` command will guide the dev through the process of creating a changelog and updating the package.json version. 

https://github.com/user-attachments/assets/33bc3497-c85c-4123-9d6c-82686ba12917